CHANGELOG.md

```diff
@@ -48,6 +50,7 @@
         name: my-config-map
         key: my-key
   ``` 
+* Cruise Control logging configuration does not contain `cruisecontrol.root.logger` property anymore. It has been replaced by `rootLogger.level`.
```

Comment:
So will this mean anyone who was running 0.21.x CC with their own logging config will need to change that when they upgrade? Or will the changes in this PR do that automatically?

Reply:
Yes, from what I understand users will need to update their logging configurations to use `log4j2` syntax when they upgrade from 0.21.x CC.